Packs/RTIR/Integrations/RTIR/RTIR.yml

```diff
@@ -90,6 +90,24 @@ script:
       name: initial-priority
     - description: Ticket final priority, given as number in the range 0-100.
       name: final-priority
+    - description: Ticket MemberOF links.
+      isArray: true
+      name: member-of
+    - description: Ticket Members links.
```

Comment:
The input should be a link? i.e. a url? not an ID?

Reply:
ticket number/id

Reply:
in that case let's fic the description it says `Ticket MemberOF links.` which is confusing, probably needs to be `Ticket MemberOF ID` etc...